CHANGELOG.md

```diff
@@ -13,6 +13,20 @@ This changelog documents all notable user-facing changes of VAST.
 
 ## Unreleased
 
+### ⚡️ Breaking Changes
```

Comment:
Uh, was this added intentionally?

Reply:
Nope, that was definitely a mistake. Can you remove this in your other PR that's likely next to be merged?

Reply:
The one for the partition_selector is probably next, but that doesn't touch the changelog. I'll remove it in the PR that moves around the options.

Reply:
Thanks!